File: fpsviewer-no-op/src/main/java/com/silencedut/fpsviewer/IViewer.java
package com.silencedut.fpsviewer;

import android.app.Application;

/**
 * @author SilenceDut
 * @date 2019-05-21
 */
public interface IViewer {
    void initViewer(Application application, FpsConfig fpsConfig);
    FpsConfig fpsConfig();
    void appendSection(String sectionName);
    void removeSection(String sectionName);
}

File: fpsviewer-no-op/src/main/java/com/silencedut/fpsviewer/IViewer.kt
package com.silencedut.fpsviewer

import android.app.Application

interface IViewer  {
    fun initViewer(application: Application, fpsConfig: FpsConfig? = null)
    fun fpsConfig():FpsConfig
    fun appendSection(sectionName:String)
    fun removeSection(sectionName:String)
}